internal/ecrconsumer/image_values_test.go

```diff
@@ -280,11 +286,20 @@ func TestWithImage(t *testing.T) {
 			Repository: "oof",
 			Tag:        "new-tag",
 		}
-		outputRls := WithImage(expectedImg, rls)
-		assert.Exactly(t, rls, outputRls)
+		outputRls, err := WithImage(expectedImg, rls)
+		assert.NoError(t, err)
+		inputValues := getChartValues(rls)
+		outputValues := getChartValues(outputRls)
+		assert.Exactly(t, inputValues, outputValues)
 	})
 }
 
+func getChartValues(r shipitv1beta1.HelmRelease) map[string]interface{} {
```

Comment:
Let's move this logic (and the duplicated json code in `WithImage`) to a method on `HelmRelease`

Reply:
I don't think `json.Unmarshal` will do the right thing when `Spec.Values.Raw` is yaml: https://play.golang.org/p/KOju4VP_Yfw

Reply:
Why would it be YAML? The API is going to Marshal whatever we put in the `values` field to JSON.

Reply:
We verified that `Values.Raw` is serialized json, so this code works.